administration/src/components/regions/RegionOverview.tsx

```diff
@@ -0,0 +1,102 @@
+import { Button, Card, H4, TextArea } from '@blueprintjs/core'
+import React, { ReactElement, useState } from 'react'
+import styled from 'styled-components'
+import { useUpdateDataPolicyMutation } from '../../generated/graphql'
+import { useAppToaster } from '../AppToaster'
+
+const Content = styled.div`
+  padding: 0 6rem;
+  width: 100%;
+  z-index: 0;
+  flex-grow: 1;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  align-items: center;
+`
```

Comment:
This looks quite odd on "larger" screen resolutions (like my laptop has).
I think we need some `flex-direction: column` in `Content` to force the correct layout

Reply:
i adjusted the style and added some margin and bigger headline to improve the layout